scripts/i18n-helper-effects.js

```diff
@@ -0,0 +1,55 @@
+export default class I18nHelperEffects {
+
+
+  constructor() {
+    // If midi-qol module is present and active
+    this.useMidiQol = game.modules.get('midi-qol')?.active;
```

Comment:
This should be a private getter.

```js
get _usesMidiQol() {
  return game.modules.get('midi-qol')?.active;
}
```

Reply:
Then you should just use the private getter everywhere and not set the boolean on the constructor

Reply:
No problem